.dockerignore

```diff
@@ -16,6 +16,7 @@ scripts/
 !scripts/test.sh
 !scripts/configure.sh
 !scripts/test-transaction.sh
+!scripts/wait-for-it.sh
```

Comment:
This script is used in the docker-compose, but by it being excluded by the .dockerignore prevented it from being available there.